src/modules/swap/handlers/handlerSwap.js

```diff
@@ -76,7 +76,7 @@ class Swap {
   }
   getQuotesForSet(arr) {
     const quotes = [];
-    const provider = this.providers[2];
+    const provider = this.providers[3];
```

Comment:
Changing provider from paraswap to changelly, since changelly has a rate for '0xbtc'